shared/prometheus/service.go

```diff
@@ -92,5 +94,9 @@ func (s *Service) Stop() error {
 // Status always returns nil.
 // TODO(1207): Add service health checks.
 func (s *Service) Status() error {
+	if s.failStatus != nil {
+		log.Errorf("Prometheus service is unhealthy : %s", s.server.Addr)
```

Comment:
Please remove this. The `Status()` method is called many times and this will bloat the logs

Reply:
You mean remove line 98 or both lines (97 and 98)?